Strategy11/formidable-forms
# Duplicate unsaved field opts

Fix https://github.com/Strategy11/formidable-pro/issues/3726

This PR extends to making sure field options are synced when a field is duplicated for Checkbox, Radio and Dropdown fields.

### Test steps
1. Create a form and add a Checkbox, Radio and Dropdown fields
2. Add/modify/remove options from each of those fields as needed.
3. Duplicate the fields
4. Check that the duplicate field options are in sync with the original field.
